.github/workflows/rust.yml

```diff
@@ -114,3 +114,44 @@ jobs:
         with:
           # When retry is implemented, we can enable it again: https://github.com/codecov/codecov-action/issues/926
           fail_ci_if_error: false
+  demo-rollup-local:
+    runs-on: ubuntu-latest
+    env:
+      SCCACHE_GHA_ENABLED: "true"
+      RUSTC_WRAPPER: "sccache"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.3
+        with:
+          version: "v0.4.0"
```

Comment:
Nit: in latest main I've removed these 2 lines, so there's always latest sccache